libs/accounts/src/lib/accounts-data-provider.ts

```diff
@@ -52,17 +53,18 @@ export const getId = (
 ) => `${data.type}-${data.asset.symbol}-${data.market?.id ?? 'null'}`;
 
 const update = (
-  draft: Accounts_party_accounts[],
+  data: Accounts_party_accounts[],
   delta: AccountSubscribe_accounts
-) => {
-  const id = getId(delta);
-  const index = draft.findIndex((a) => getId(a) === id);
-  if (index !== -1) {
-    draft[index] = delta;
-  } else {
-    draft.push(delta);
-  }
-};
+) =>
+  produce(data, (draft) => {
```

Comment:
question (non-blocking): Same here, why don't we have to do anything with the result of produce?